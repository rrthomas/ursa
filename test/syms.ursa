#!/usr/bin/env ursa
// List symbols in input

use jslib.fs
let s = fs.readFileSync(argv[1], "utf-8")
let regex = RegExp("\\p{L}+", "ug")

loop {
    let m = regex.exec(s)
    if m == null { break }
    print(m[0])
}
