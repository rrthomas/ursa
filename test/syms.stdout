use
jslib
fs
fs
writeSync
foo
is
stdout